### PR TITLE
Nix fixes and CTL 5.0

### LINKED ISSUE #3: Scaffold repository
Scaffold repository.
Current requirements are:
- CTL for off-chain
- PlutusTx and Haskell for on-chain

Acceptance criteria: 
- [ ] simple contract to pay from PKH to validator script can be executed

### PR DESCRIPTION
**Pull request by mikekeke**
Closes #3 

This PR updates CTL to 5.0 and fixes Nix app for CTL runtime to be able to configure it.

### CODE REVIEW

File: off-chain/test/Examples/AdditionExample.purs [hunk 1]
```diff
@@ -45,9 +45,9 @@ instance ToData AdditionRedeemer where
   toData (AdditionRedeemer {increaseAmount}) = toData increaseAmount
 
 action2UTxO :: UserAction AdditionAction -> TransactionHash -> Transaction
-action2UTxO userAction useUTxO = 
-  case (unwrap userAction).action of 
-      AddAmount amount -> undefined -- TODO create a transaction consuming useUTxO that 
+action2UTxO userAction useUTxO = undefined
+--  case (unwrap userAction).action of
```
Comment (mikekeke) [inline off-chain/test/Examples/AdditionExample.purs:49]: Commented out this part to make compilation and Nix check pass.

File: off-chain/test/E2E.purs [hunk 2]
```diff
@@ -1,53 +1,54 @@
 -- | This module is used to serve the E2E tests to the headless browser.
 module Seath.Test.E2E.Serve where
 
-import Contract.Prelude
```
Comment (mikekeke) [inline off-chain/test/E2E.purs:4]: Just commented out everything here for now to make build and check pass. Do we need it? Maybe delete for now.

File: off-chain/package.json [hunk 3]
```diff
@@ -7,6 +7,7 @@
     "test": "test"
   },
   "scripts": {
+    "blockfrost-test": "source ./test/blockfrost.env && spago run --main Scaffold.Test.Blockfrost",
```
Comment (Luis-omega) [inline off-chain/package.json:10]: Are we planning to use blockfrost as backend?

File: off-chain/package.json [hunk 4]
```diff
@@ -7,6 +7,7 @@
...
     "test": "spago run --main Test.Seath.Main",
     "e2e-serve": "make e2e-serve",
```
Comment (Luis-omega) [inline off-chain/package.json:12]: we probably can delete all the e2e tests from ctl, at least this part won't work now that `makefile` is deleted.

File: off-chain/test/E2E.purs (hunk 2 above)
Comment (Luis-omega) [reply off-chain/test/E2E.purs:4]: Those are tests inherited from CTL, we don't need them but we definitively can reuse this and fill it with our own test, so, leaving them commented, for now, is the way to go I think.

File: off-chain/test/Examples/AdditionExample.purs (hunk 1 above)
Comment (Luis-omega) [reply off-chain/test/Examples/AdditionExample.purs:49]: The work on this would be on another pr, so, it's ok.

**Pull request merged**